src/gui/dbsettings/DatabaseSettingsWidgetGeneral.cpp

```diff
@@ -130,6 +138,16 @@ bool DatabaseSettingsWidgetGeneral::save()
         truncate = true;
     }
 
+    int autosaveDelay;
```

Comment:
Will it be more compact?

```c++
    int autosaveDelay = 0;
    if (m_ui->autosaveDelayCheckBox->isChecked()) {
        autosaveDelay = m_ui->autosaveDelaySpinBox->value();
    }
```

Reply:
Looks good, made changes at [2e5c826](https://github.com/keepassxreboot/keepassxc/pull/9100/commits/2e5c82606a58ecbbd3ea409c7208e3c036e08d46)